### PR TITLE
Corrected incorrect translation 

### PR DESCRIPTION
**Pull request by sbzlzh**

### CODE REVIEW

**Review by TimGoll — APPROVED** [review]
You don't have to close and reopen your pullrequest. You can just edit the last one. But this one is good now.

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,9 +13,6 @@ All notable changes to TTT2 will be documented here. Inspired by [keep a changel
   - data.selectValue is added, use it instead of data.selectName to choose the value you set
   - data.selectTitle is added and shall replace data.selectName
 - New setting to disable session limits entirely. (by @Reispfannenfresser)
-- Added `GM:TTT2AdminCheck` hook
-  - Replaced all `IsSuperAdmin()` checks with this hook
-  - This hook can be used to allow custom usergroups through these checks
```
Comment (Histalek) [inline CHANGELOG.md:18]: Could you please re-add these lines?

**Review by Histalek — CHANGES_REQUESTED** [review]
Besides my comment on the changelog this looks good

File: CHANGELOG.md (hunk 1 above)
Comment (sbzlzh) [reply CHANGELOG.md:18]: ok

Comment (sbzlzh) [reply CHANGELOG.md]: Now it has been added

**Review by Histalek — APPROVED** [review]

**Pull request merged**